tests/suites/test_suite_ssl.function

```diff
@@ -2648,3 +2915,64 @@ void ssl_session_serialize_version_check( int corrupt_major,
 
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_X509_CRT_PARSE_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_RSA_C:MBEDTLS_ECDSA_C:MBEDTLS_ECP_DP_SECP256R1_ENABLED:!MBEDTLS_USE_PSA_CRYPTO */
```

Comment:
Does this case really depend on this many defines?

Reply:
It's looking that way from my perspective. To went through all handshake I am using the same code as it is used in programs ssl_server2 and ssl_client2 with default configuration. That configuration is quite specific because it uses hardcoded certificates and etc. In ssl_server2 and ssl_client2 it is pretty easy to use custom configuration by setting parameters during running those programs. Unfortunately current code used in test suites is not that generic and it is more sensitive for changes in the configuration of mbedtls. In the future it would be nice to add parameterization of different certificates, ciphersuites, version, etc. but currently we are focusing only on memory optimalization, aren't we?